{"name": "verifying search box is present in Google home page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: was killed\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.17134 x86_64)\n\n", "trace": "  File \"c:\\program files\\python36\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\program files\\python36\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Defination.py\", line 26, in I_open_a_browser\n    d.openBrowser()\n  File \"C:\\Users\\Ramesh-Shwetha\\PycharmProjects\\True_Fit_Automation_Framework\\utility\\WrapperDriver.py\", line 36, in openBrowser\n    self.driver = webdriver.Chrome(executable_path=config['DEFAULT']['chromedriverpath'])\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 81, in __init__\n    desired_capabilities=desired_capabilities)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I open a browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: was killed\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.17134 x86_64)\n\n", "trace": "  File \"c:\\program files\\python36\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\program files\\python36\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Defination.py\", line 26, in I_open_a_browser\n    d.openBrowser()\n  File \"C:\\Users\\Ramesh-Shwetha\\PycharmProjects\\True_Fit_Automation_Framework\\utility\\WrapperDriver.py\", line 36, in openBrowser\n    self.driver = webdriver.Chrome(executable_path=config['DEFAULT']['chromedriverpath'])\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 81, in __init__\n    desired_capabilities=desired_capabilities)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\program files\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1540335766187, "stop": 1540335770373}, {"name": "When I am loading google url as \"http://www.google.com\"", "status": "skipped", "start": 1540335770373, "stop": 1540335770373}, {"name": "Then I am making sure that search box is present", "status": "skipped", "start": 1540335770373, "stop": 1540335770373}, {"name": "And I am closing the browser", "status": "skipped", "start": 1540335770373, "stop": 1540335770373}], "start": 1540335766183, "stop": 1540335770373, "uuid": "25ee8c0c-231c-46bf-9730-e1504cad9ea8", "historyId": "203bb1ead99c875c6ca3985a43710e9a", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Testing True Fit"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}